modules/simple-payments/simple-payments.css

```diff
@@ -0,0 +1,17 @@
+.jetpack-simple-payments__purchase-message {
+	display: none;
+	color: #fff;
```

Comment:
Did it work? `jetpack-simple-payments__purchase-message` is a `<p>` isn't it? I've asked to change it to `<p>` since most themes should have some spacing for it so that we don't have to add some arbitrary margin.